src/cli/cli.cpp

```diff
@@ -2091,6 +2091,35 @@ otError Interpreter::ProcessIpAddr(Arg aArgs[])
         {
             OutputIp6AddressLine(*otThreadGetMeshLocalEid(GetInstancePtr()));
         }
+        else if (aArgs[0] == "-v")
+        {
+            const otNetifAddress *unicastAddrs = otIp6GetUnicastAddresses(GetInstancePtr());
+
+            for (const otNetifAddress *addr = unicastAddrs; addr; addr = addr->mNext)
+            {
+                OutputIp6Address(addr->mAddress);
+                OutputFormat(" Origin:");
+                switch (addr->mAddressOrigin)
+                {
+                case OT_ADDRESS_ORIGIN_SLAAC:
+                    OutputFormat(" %s", "slaac");
```

Comment:
```suggestion
                    OutputFormat("%s", "slaac");
```
removing space so that the output would be like `origin:slaac` ?
It may help parser to interpret the command output.